TestingTool/actions.py

```diff
@@ -41,10 +41,19 @@ def input_element(action_args, browser_instance):
     data.sendKeys(input_text)
     return True, {"success": "Element from xpath took the received value"}
 
+def go_to_url(action_args, browser_instance):
+    result, driver = element_from_xpath(action_args, browser_instance)
+    go_to_url = action_args.get("go_to_url")
```

Comment:
redenumeste te rog argumentul in 'url' simplu, e mai scurt si mai concis, e obvious ce reprezinta :)